includes/payments/functions.php

```diff
@@ -161,7 +161,7 @@ function edd_insert_payment( $payment_data = array() ) {
 			'currency'     => $payment_data['currency'],
 			'downloads'    => $payment_data['downloads'],
 			'user_info'    => $payment_data['user_info'],
-			'cart_details' => $payment_data['cart_details']
+			'cart_details' => $payment_data['cart_details'],
```

Comment:
@cklosowski I thought EDD followed [WP coding standards](https://make.wordpress.org/core/handbook/coding-standards/php/#indentation), so I left the trailing coming when reverting the first commit.